experiments/langchain_lab/scripts/doc_loader.py

```diff
@@ -0,0 +1,41 @@
+"""Script to create vector DB for usage of RAG.
+
+Reference:
+  - [Document Loaders in LangChain]()
+"""
+
+import dotenv
+from langchain.document_loaders.csv_loader import CSVLoader
+from langchain_community.document_loaders import TextLoader
+from langchain_community.vectorstores import Chroma
+from langchain_community.vectorstores import FAISS
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain_community.embeddings import OpenAIEmbeddings
+
+
+PERSISTENT_CHROMA_PATH = "chroma_data"
+dotenv.load_dotenv()
```

Comment:
Are there some env vars that needs to be loaded via dotenv?

Reply:
Good question. Only `OPENAI_API_KEY` (Open API key) is required. I have added comment nearby for reference.